bundles/org.openhab.ui.dashboard/README.md

```diff
@@ -0,0 +1,34 @@
+# Dashboard
```

Comment:
@ThomDietrich Any suggestion where this page might best fit in on docs.openhab.org and whether we should automatically pick the page up from here, or simply copy the content over to openhab-docs?